dataplane/vppagent/pkg/vppagent/metrics_collector.go

```diff
@@ -33,7 +34,7 @@ func (m *MetricsCollector) CollectAsync(monitor metrics.MetricsMonitor, endpoint
 }
 
 func (m *MetricsCollector) collect(monitor metrics.MetricsMonitor, endpoint string) {
-	conn, err := tools.DialTCP(endpoint)
+	conn, err := grpc.Dial(endpoint, grpc.WithInsecure())
```

Comment:
Have we not secured the Monitor calls yet?

Reply:
This is call to `vpp-agent`, by default it starts `grpc.Server` with insecure mode. Probably we can configure it to work with security. If so I will add it in the next PR